lawlibrary/urls.py

```diff
@@ -1,14 +1,15 @@
 from django.urls import include, path
 
 from lawlibrary import views
+from liiweb.views import LocalityLegislationListView
 
 urlpatterns = [
     path("", views.HomePageView.as_view(), name="home_page"),
     path("legislation/", views.LegislationListView.as_view(), name="legislation_list"),
     path(
-        "legislation/provincial",
-        views.ProvincialLegislationView.as_view(),
-        name="provincial_legislation",
+        "legislation/locality",
```

Comment:
I think we should keep the existing URL of `legislation/provincial` because it's semantically meaningful.